fs2-stream/src/test/scala/neotypes/fs2/Fs2Suite.scala

```diff
@@ -20,6 +20,8 @@ object Fs2Testkit extends StreamTestkit[Fs2IoStream, IO](IOTestkit) {
 
       override final val streamInstance: Stream.Aux[Fs2IoStream, IO] =
         implicitly
+
+      override def streamConcurrently(stream1: Fs2IoStream[Unit], stream2: Fs2IoStream[Unit]): Fs2IoStream[Unit] = stream1.concurrently(stream2)
```

Comment:
Looking at the [documentation of **fs2**](https://oss.sonatype.org/service/local/repositories/releases/archive/co/fs2/fs2-core_2.12/3.0.4/fs2-core_2.12-3.0.4-javadoc.jar/!/fs2/Stream$.html) it seems it would be better to use `merge` over `concurrently`, since the former ensures both `Streams` are finished.

Also, please move the implementation to the next line :)